docs/resources/kubernetes_cloud.md

```diff
@@ -37,6 +37,7 @@ resource "juju_model" "my-model" {
 ### Optional
 
 - `kubernetes_config` (String, Sensitive) The kubernetes config file path for the cloud. Cloud credentials will be added to the Juju controller for you.
+- `no_service_account` (Boolean) If set to true, the k8s cloud will not create a service account within the k8s cluster and override the authentication info in the k8s config. In this way we avoid reaching for the k8s API server when adding a k8s cloud.
```

Comment:
Some ideas for naming courtesy of gpt, "skip_service_account", "disable_service_account", "use_existing_auth", "preserve_kubeconfig_auth", "skip_service_account_creation".